paddle/scripts/submit_local.sh.in

```diff
@@ -112,13 +112,13 @@ fi
 
 case "$1" in
     "train")
-        ${DEBUGGER} $MYDIR/../opt/paddle/bin/paddle_trainer ${@:2}
+        ${DEBUGGER} $MYDIR/../../opt/paddle/bin/paddle_trainer ${@:2}
```

Comment:
If install with `pip install`, the `paddle` script will be under `/usr/local/bin/paddle` but binaries are under somewhere like: `/usr/lib/python2.7/site-packages/usr/opt/paddle/bin`.

So the correct way is to check both where `paddle` installed and the python package installed, and find where the binaries are.